docs/deployment_guide/partner_editable/architecture.adoc

```diff
@@ -20,11 +20,11 @@ As shown in <<architecture1>>, this Quick Start sets up the following:
 *** An Amazon Elastic Compute Cloud (Amazon EC2) Linux instance to bootstrap the installation of VMware TAP in an Amazon EKS cluster and serve as administrator bastion host.
 *** An Amazon EC2 Windows bastion host for access to the Tanzu Application Platform user interface.
 * In the private subnets:
-** A group of Amazon EKS nodes in an Auto Scaling Group with Contour Envoy ingress controller pods and the TAP installation. For a complete list of the {partner-product-short-name} packages, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.2/tap/GUID-install-components.html#verify-the-installed-packages-1[Verify the installed packages].
+** A group of Amazon EKS nodes in an Auto Scaling Group with Contour Envoy ingress controller pods and the TAP installation. For a complete list of the {partner-product-short-name} packages, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-install-components.html#verify-the-installed-packages-1[Verify the installed packages].
```

Comment:
```suggestion
** A group of Amazon EKS nodes in an Auto Scaling Group with Contour Envoy inbound controller pods and the TAP installation. For a complete list of the {partner-product-short-name} packages, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-install-components.html#verify-the-installed-packages-1[Verify the installed packages].
```

Reply:
"Ingress controller" is a specific & intentional technical term for this architecture ([reference](https://projectcontour.io/)).